ert/data/record/_record.py

```diff
@@ -59,6 +59,14 @@ class RecordType(str, Enum):
 
 
 class Record(ABC):
+    """The :class:`Record` class is an abstract class that
+    includes record data and the record type. It represents a
+    basic data `unit` to accommodate input and output to stages.
```

Comment:
It is used for a bit more than input and output to stages, so maybe we could simply put:
```suggestion
    includes record data and the record type. It represents the
    basic data `unit` used in ert.
```